### DISCUSSION TITLE
org.apache.skywalking.oap.server.core.analysis.worker.MetricsPersistentWorker -47735341 [pool-5-thread-1] ERROR [] - Can't split endpoint id into 2 parts, c3RhY2s=.1_

### DISCUSSION THREAD

**Discussion by njupt-tj** [apache/skywalking#8189] [Q&A]
![image](https://user-images.githubusercontent.com/27984465/143601358-fcb74464-4c9c-4e9e-a2ca-f162a350209f.png)
How to solve this problem？？？

**Comment by wu-sheng** [comment]
You are deploying the 8.x OAP server on the data generated by 6.x or 7.x
